src/Python/tests/tests.py

```diff
@@ -60,18 +69,23 @@ def test_unit_combos(self):
         MeasurementUnit.create('m^2') == MeasurementUnit.create('metres^2')
         MeasurementUnit.create('C') == MeasurementUnit.create('celsius')
         MeasurementUnit.create('m/s') == MeasurementUnit.create('metres/sec')
-        MeasurementUnit.create('mm/s^2') == MeasurementUnit.create('millimeter/sec^2')
+        MeasurementUnit.create(
+            'mm/s^2') == MeasurementUnit.create('millimeter/sec^2')
```

Comment:
Not a fan of this either.  You break in the middle of a block.  It makes it seem like the `==` is part of the `create` call, not a comparison.